dual_net.py

```diff
@@ -279,7 +268,7 @@ def bootstrap(working_dir, **hparams):
     sess = tf.Session(graph=tf.Graph())
     with sess.graph.as_default():
         features, labels = get_inference_input()
-        model_fn(features, labels, tf.estimator.ModeKeys.PREDICT, hparams)
+        model_fn(features, labels, tf.estimator.ModeKeys.PREDICT)
```

Comment:
so we're sure nothing was actually being set by the hparams here?

Reply:
Well, they were but now it's being set by flags.